scripts/merge-script/strategies/strategy.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Config} from '../config';
+import {GitClient} from '../git';
+import {PullRequest, PullRequestFailure} from '../pull-request';
+
+/** Name of a temporary branch that contains the head of a currently-processed PR. */
+export const TEMP_PR_HEAD_BRANCH = 'merge_pr_head';
```

Comment:
Should this branch name be unique?  Or is it okay that it gets reused?

Reply:
It can be any arbitrary branch name. This is the one from the framework's merge script.

Reply:
Tried to make this more clear in the description.